Category/categoryRouter.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+
+const Category = require('./Category.js');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+    Category.find().select('title -_id')
+    .then((categories) => {
+        res.status(200).json(categories)
+    })
+    .catch((error) => {
+        res.status(500).json(error)
+    })
+})
+
+router.post('/', (req, res) => {
+    const {title } = req.body
```

Comment:
Character spacing here is a bit off. According to Airbnb's styleguide (https://github.com/airbnb/javascript#whitespace--in-braces), there should be spaces inside curly braces.